src/ocm/components/clusterDetail/ClusterProperties.tsx

```diff
@@ -12,24 +12,25 @@ import {
   selectIpv6Cidr,
   selectIpv4HostPrefix,
   selectIpv6HostPrefix,
+  useFeatureSupportLevel,
 } from '../../../common';
 import { useTranslation } from '../../../common/hooks/use-translation-wrapper';
 import { ClusterFeatureSupportLevelsDetailItem } from '../featureSupportLevels';
 import OpenShiftVersionDetail from './OpenShiftVersionDetail';
 
-const CpuArchTitle = () => (
-  <>
-    {'CPU architecture '}
-    <PopoverIcon
-      bodyContent={
-        <p>
-          The generated ISO is specific to the cluster’s CPU architecture. Only hosts with the same
-          CPU architecture can be added to this cluster.
-        </p>
-      }
-    />
-  </>
-);
+const getCpuArchTitle = (isMultiArchSupported: boolean) => {
```

Comment:
Why not keep this as a React component and pass it the `isMultiArchSupported` prop?